specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -12806,6 +15566,90 @@
         }
       }
     },
+    "ContinuousWebJob": {
+      "description": "Continuous Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ContinuousWebJob resource specific properties",
+          "properties": {
+            "status": {
```

Comment:
Decorate with something akin to: 
[PropertySettings(AllowedStringValues = new[] { "Value1", "Value2", "Value3" }, ModelValuesAsString = true, AllowedStringValueCollectionName = "WebJobStatus")]

---
In reply to: [134406282](https://github.com/Azure/azure-rest-api-specs/pull/1551#discussion_r134406282) [](ancestors = 134406282)